app/services/invoices/progressive_billing_service.rb

```diff
@@ -22,19 +22,10 @@ def call
         Credits::ProgressiveBillingService.call(invoice:)
         Credits::AppliedCouponsService.call(invoice:)
 
-        if customer_provider_taxation?
-          taxes_result = Integrations::Aggregator::Taxes::Invoices::CreateService.call(invoice:, fees: invoice.fees)
+        totals_result = Invoices::ComputeTaxesAndTotalsService.call(invoice:)
+        return totals_result if !totals_result.success? && totals_result.error.is_a?(BaseService::UnknownTaxFailure)
```

Comment:
Maybe naming is confusing, but I couldn't find better one:) The idea is:
- if unknown error happens, we don't want to raise error since it would rollback all DB changes
- instead we just want to catch this state and leave invoice in `pending` status
- Async job for fetching taxes will process `pending` invoice and put it either in `finalized` or `failed` (if there are tax errors like invalid product mapping)